lib/message_bus.rb

```diff
@@ -554,6 +554,25 @@ def keepalive_interval
     @config[:keepalive_interval] || 60
   end
 
+  # Registers a message filter that allows messages to be filtered from the client.
+  #
+  # @param [Regexp] a regular expression to match against a message's channel
+  #
+  # @yieldparam [MessageBus::Message] message published to the channel that matched the regular expression
+  # @yieldreturn [Boolean] whether the message should be published to the client
+  # @return [void]
+  def register_message_filter(regexp, &blk)
```

Comment:
maybe we don't need a regex ... start_with with a string should be good enough and a touch faster anyway.

`register_client_message_filter(channel_prefix, &blk)`